File: 09_week_1/popFront.js
/*	popFront	*/
function popFront(arr) {
	var first_val = arr[0];

	for (var i = 0, j = arr.length-1; i < j; i++) {
		arr[i] = arr[i+1];
	}
	arr.pop();

	return first_val;
}

var arr = [1,2,3,4];
console.log(popFront(arr));
console.log(arr);

File: 09_week_1/popFront.py
#popFront
def popFront(arr):

	first_val = arr[0]

	for idx in range(1, len(arr)):
		arr[idx-1] = arr[idx]

	arr.pop() # funny how this takes an argument (index) to be popped!

	return first_val

arr = [1, 2, 3, 4, "Hello", "Hi", 5]
print(popFront(arr))
print(arr)
